api/v3/indexercluster_types.go

```diff
@@ -123,7 +124,6 @@ type IndexerClusterStatus struct {
 // +kubebuilder:printcolumn:name="Desired",type="integer",JSONPath=".status.replicas",description="Desired number of indexer peers"
 // +kubebuilder:printcolumn:name="Ready",type="integer",JSONPath=".status.readyReplicas",description="Current number of ready indexer peers"
 // +kubebuilder:printcolumn:name="Age",type="date",JSONPath=".metadata.creationTimestamp",description="Age of indexer cluster"
-// +kubebuilder:storageversion
```

Comment:
why this change?

Reply:
we are not going to store v3 instead we will store v4. we only server v3

Reply:
Each CRD can be stored once. So for the CRDs that are in both V4 and V3, such as IndexerCluster, we only add the storage option in the latest version(V4).